<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSchemePresetWidget</class>
 <widget class="QWidget" name="ColorSchemePresetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>167</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>271</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Color Scheme Presets</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="typesLabel">
     <property name="text">
      <string>&amp;Types Filter:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>divergingCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="deficiencyLabel">
     <property name="text">
      <string>De&amp;ficiency:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>deficiencyComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="classesLabel">
     <property name="text">
      <string>&amp;Classes Filter:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>classesSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="deficiencyComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="classesSpinBox"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="widgetzeug::ColorSchemeGraphicsView" name="graphicsView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="interactive">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="divergingCheckBox">
       <property name="text">
        <string>&amp;diverging</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="sequentialCheckBox">
       <property name="text">
        <string>&amp;sequential</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="qualitativeCheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;qualitative</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>widgetzeug::ColorSchemeGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>ColorSchemeGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>classesSpinBox</tabstop>
  <tabstop>deficiencyComboBox</tabstop>
  <tabstop>graphicsView</tabstop>
  <tabstop>divergingCheckBox</tabstop>
  <tabstop>sequentialCheckBox</tabstop>
  <tabstop>qualitativeCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
